### PR TITLE
doc: reformat health-messages.rst and revise SLOW_OPS section

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by batrick — APPROVED** [review]

**Review by neha-ojha — APPROVED** [review]

File: doc/rados/operations/health-checks.rst [hunk 1]
```diff
@@ -1015,12 +1015,12 @@ told to roll back to a previous version of the object. See
...
 ________
 
-One or more OSD requests is taking a long time to process.  This can
+One or more OSD or monitor requests is taking a long time to process.  This can
 be an indication of extreme load, a slow storage device, or a software
 bug.
 
-The request queue on the OSD(s) in question can be queried with the
-following command, executed from the OSD host::
+The request queue on the daemon in question can be queried with the
```
Comment (zdover23) [inline doc/rados/operations/health-checks.rst:1022]: s/on the daemon/for the daemon/

I think in this case, "for" is the preposition that fits better.

**Review by zdover23 — CHANGES_REQUESTED** [review]
I only have one request for a change to this text.

The larger addition to the text could no doubt be improved, but I think after a first readthrough that it's good enough to go into the docs.

**Review by jdurgin — APPROVED** [review]

File: doc/rados/operations/health-checks.rst (hunk 1 above)
Comment (tchaikov) [reply doc/rados/operations/health-checks.rst]: @zdover23 updated and repushed.

**Comment by tchaikov** [comment]
@zdover23 ping?

**Comment by badone** [comment]
looks like the make check failure is https://tracker.ceph.com/issues/48108

**Comment by tchaikov** [comment]
jenkins test make check

**Review by zdover23 — APPROVED** [review]

**Pull request merged**